bitrise.yml

```diff
@@ -204,10 +213,12 @@ workflows:
             # !/bin/env bash
             set -ex
             echo "test" > $TMP_DIR/File.txt
-    - cache-push:
+    - git::https://github.com/bitrise-steplib/steps-cache-push@STEP-1996-arch:
```

Comment:
ToDo: Remove when released.

Reply:
Can you release the push step first and then update this PR before merging. Just to be sure we do not forget about this. 😄